packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -68,16 +68,22 @@ class FileCollection extends DocumentCollection {
    * async findReferencedBy - Returns the list of files referenced by a document — see https://docs.cozy.io/en/cozy-stack/references-docs-in-vfs/
    *
    * @param  {object} document     A JSON representing a document, with at least a `_type` and `_id` field.
-   * @param  {number} { skip = 0   For pagination, the number of referenced files to skip
-   * @param  {number} limit } For pagination, the number of results to return.
+   * @param  {number} skip    For skip-based pagination, the number of referenced files to skip.
+   * @param  {number} limit For pagination, the number of results to return.
+   * @param  {object} cursor   For cursor-based pagination, the index cursor.
```

Comment:
options.skip
options.limit
options.cursor

Reading the doc I was confused, I thought there were 4 arguments to the function.